_data/collections.yml

```
@@ -15,4 +15,9 @@
   url: https://longplay.rocks/data/collections/NY100J.json
   count: 100
   date: 2008-05-19
+- name: "Rolling Stone: Concept Albums"
+  article_url: https://www.rollingstone.com/music/music-lists/best-concept-albums-1234604040/the-roots-20-1234604517/
```

```suggestion
  article_url: https://www.rollingstone.com/music/music-lists/best-concept-albums-1234604040/
```

The link was the the bottom of the page. Better to link to just the page.